apps/rush-lib/src/examples/Legacy_CustomCommandLine.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Add a code comment at the top of this file saying something like this:

// This file recreates the command-line options that were built into Rush prior to version 4.
// Note that earlier versions of Rush also performed the following transformations to
// produce the command-line that was passed to the NPM scripts in the project's
// package.json file:
//
// - If "--vso" was passed to Rush, then "--color" was passed to the script instead
// - Before running a "build" script, the "clean" script was automatically executed first
//
// Rush 4 no longer performs these sorts of transformations.  Instead, it aims for
// the Rush options and script options to be as similar as possible.

Reply:
(and maybe we don't need the "--color" in this example)

---
In reply to: [147541732](https://github.com/Microsoft/web-build-tools/pull/387#discussion_r147541732) [](ancestors = 147541732)